src/locale/fr-FR.h

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 - Mathieu D(@Knackie)
+// Locale/language file for French / Quebec.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_FR_FR_H_
+#define LOCALE_FR_FR_H_
```

Comment:
Technically, it would be `LOCALE_FR_CA_H_` for Québécois?
The filename should also be changed too. i.e. `src/locale/fr-CA.h`

I'm not certain so I will defer to you, but I have had some French friends who said there are differences between French/France vs French/Canada(Quebec).

Reply:
for this word, this is the same in Canada and French